config/schema/base_elasticsearch_type.json

```diff
@@ -33,6 +33,8 @@
     "title",
     "topic_content_ids",
     "updated_at",
-    "user_journey_document_supertype"
+    "user_journey_document_supertype",
+    "content_purpose_supergroup",
```

Comment:
It's a minor thing, but I'd put these in alphabetical order.

Reply:
Ok cool. Cheers

Reply:
It all looks really good!